### PR TITLE
fix: collections active state

### PR DESCRIPTION
**Pull request by seaerchin**
## Problem
we were previously using the wrong property, leading to collections not having the correct state in side bar when we are already in the collection

## Solution
update to correct property 

## Checks
1. Click into a collection
2. the sidebar should be highlighted for the collection

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -33842,7 +33842,7 @@
         "eslint-plugin-import": "^2.30.0",
         "eslint-plugin-jsx-a11y": "^6.10.0",
         "eslint-plugin-react": "^7.36.1",
-        "eslint-plugin-react-hooks": "*",
+        "eslint-plugin-react-hooks": "beta",
```
Comment (harishv7) [inline package-lock.json:33845]: is this change intended?

**Review by harishv7 — APPROVED** [review]
Approved with 1 minor comment

File: package-lock.json (hunk 1 above)
Comment (seaerchin) [reply package-lock.json:33845]: i did an `npm i` and this got updated - i'll change it back in the interest of safety if you're concerned!
